src/combobox/combobox.component.ts

```diff
@@ -285,6 +285,12 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 						this.showClearButton = true;
 						this.selectedValue = event.item.content;
 						this.propagateChangeCallback(event.item);
+						// Under some circumstances the view does not get updated with the selected
+						// item for a single selection combobox. This manually updates the input with the
+						// correct selected item.
+						if (this.view.getSelected()[0]) {
+							this.input.nativeElement.value = this.view.getSelected()[0].content;
+						}
```

Comment:
Which circumstances? Why are we bypassing `selectedValue` and the angular value update in just this one spot?

Reply:
I think it's better if I show you 😅

Reply:
Not really ... we shouldn't be updating the state of the input in two places ... either this needs to be run on the next change detection, or all the updates should go through the native element in a setter